crates/hash/src/groestl/arch/groestl_multi_avx2.rs

```diff
@@ -1,6 +1,8 @@
 // Copyright 2025 Irreducible Inc.
 
 use std::{arch::x86_64::*, array};
+
+use crate::multi_digest::ParallelMulidigestImpl;
```

Comment:
nit: typo

Reply:
You should install the pre-commit hooks, which would detect this. `pre-commit install`